src/mame/mit/cadr_tv_control.h

```diff
@@ -0,0 +1,57 @@
+// license:BSD-3-Clause
+// copyright-holders:Wilbert Pol
+#ifndef MAME_MIT_CADR_TV_CONTROL_H
+#define MAME_MIT_CADR_TV_CONTROL_H
+
+#pragma once
+
+
+#include "screen.h"
+
+
+DECLARE_DEVICE_TYPE(CADR_TV_CONTROL, cadr_tv_control_device)
+
+
+class cadr_tv_control_device : public device_t
+{
+public:
+	cadr_tv_control_device(const machine_config &mconfig, const char *tag, device_t *owner, u32 clock);
+
+	// configuration
+	auto irq_callback() { return m_irq_cb.bind(); }
+
+	u32 video_ram_read(offs_t offset);
+	void video_ram_write(offs_t offset, u32 data);
+	u32 tv_control_r(offs_t offset);
+	void tv_control_w(offs_t offset, u32 data);
+
+protected:
+	// device-level overrides
+	virtual void device_start() override ATTR_COLD;
+	virtual void device_reset() override ATTR_COLD;
+	virtual void device_add_mconfig(machine_config &config) override;
```

Comment:
Please stop copy-pasting the “device-level overrides” comment – it makes no sense.  Also, `device_add_mconfig` should be marked cold as well.